Add MySigMail

antonreshetov opened the pull request:
<!-- If you are adding new software to the list, DO NOT DELETE THE TEXT BELOW . Please make sure relevant boxes are checked [x] -->
<!-- If you are simply updating an existing entry or removing a project, DO delete the text below. -->

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/awesome-foss/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.bevry.me](https://staticsitegenerators.bevry.me/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. If login credentials are required to access the demo, please link to the credentials directly.
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` should match the platforms required to install and run the software.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

antonreshetov committed:
feat: add MySigMail project metadata

Rabenherz112 commented on software/mysigmail.yml:
```diff
@@ -0,0 +1,18 @@
+archived: false
+current_release:
+  tag: v2.0.0
+  published_at: '2025-09-18'
```
This will be added by our job. No need to manually add this.
```suggestion
```

Rabenherz112 commented on software/mysigmail.yml:
```diff
@@ -0,0 +1,18 @@
+  - AGPL-3.0
+platforms:
+  - Vue
+  - TypeScript
```
```suggestion
```

nodiscc committed:
Update software/mysigmail.yml

Co-authored-by: kokomo123 <70863536+kokomo123@users.noreply.github.com>

nodiscc committed:
Merge branch 'master' into add-mysigmail

nodiscc reviewed (commented):
* Which version from the `Plans & Pricing` section from the website, is available for self-hosting (Free/basic/Plus/Pro/Unlimited) ?

Also,

>This project is licensed under the [AGPL-3.0](https://github.com/antonreshetov/mysigmail/blob/master/LICENSE) for non-commercial use.

Meaning of "Commercial use"?

antonreshetov commented:
@nodiscc The repository represents only the client side, without the backend, so there is no concept of "Plans & Pricing" here. I also corrected the README in the License section and removed the contradictory part.

nodiscc committed:
Merge branch 'master' into add-mysigmail
